packages/datastore/src/wsmessages.ts

```diff
@@ -38,8 +38,6 @@ export namespace Collaboration {
    * Maps a msgType to the corresponding type.
    */
   export interface ITypeMap {
-    'storeid-request': StoreIdRequest;
```

Comment:
Eventually we may want to reconsider this, but I think having the storeid on page config is a good place for us to be right now.

Reply:
It's in the git history if somebody ever thinks to resurrect it. Until then it is dead code, so it should go away :)

Reply:
I am very much in favor of getting rid of dead code.

Reply:
Cool, so is this good or are you suggesting a change?

Reply:
Yeah, this is good.